pynamodb/attributes.py

```diff
@@ -64,6 +63,54 @@ def get_value(self, value):
         return value.get(serialized_dynamo_type)
 
 
+class AttributeContainer(object):
+
+    _attributes = None
+    _dynamo_to_python_attrs = None
+
+    @classmethod
+    def _initialize_attributes(cls):
+        """
+        Initialize attributes on the class.
+        """
+        cls._attributes = {}
+        cls._dynamo_to_python_attrs = {}
+        for item in dir(cls):
```

Comment:
Thank you. I noticed it later too, but it would be better to clean it up while the code is touched.